src/gourmet/gtk_extras/dialog_extras.py

```diff
@@ -671,34 +677,31 @@ def jump_to_header (self, text):
                 self.indexView.expand_row(mod.get_path(itr),True)
                 return
 
-    def parse_faq(self, filename: str) -> None:
-        """Parse file infile as our FAQ to display.
-
-        infile can be a filename or a file-like object.
-        We parse index lines according to self.INDEX_MATCHER
-        """
+    def parse_faq(self) -> None:
+        """Parse the FAQ for display."""
 
         # Clear data
         self.index_lines = []
         self.index_dic = {}
         self.text = ""
 
-        with open(filename, 'r') as fin:
-            for l in fin.readlines():
-                line = l.strip()
-                if self.INDEX_MATCHER.match(line):  # it is a heading
-                    self.index_lines.append(line)
-                    curiter = self.textbuf.get_iter_at_mark(self.textbuf.get_insert())
-                    self.index_dic[line] = self.textbuf.create_mark(None,
-                                                                    curiter,
-                                                                    left_gravity=True)
-                    self.textbuf.insert_with_tags(curiter,
-                                                  line + " ",
-                                                  self.boldtag)
-                elif line:  # it is body content
-                    self.textbuf.insert_at_cursor(line + " ")
-                else:  # an empty line is a paragraph break
-                    self.textbuf.insert_at_cursor("\n\n")
+        faq = _get_data('gourmet', 'data/FAQ').decode()
```

Comment:
```suggestion
        faq = _get_data(os.path.join('gourmet', 'data', 'FAQ')).decode()
```

Reply:
Per the [documentation](https://docs.python.org/3/library/pkgutil.html#pkgutil.get_data):
> The resource argument should be in the form of a relative filename, using / as the path separator.